cornice_swagger/swagger.py

```diff
@@ -470,8 +485,12 @@ def generate_swagger_spec(services, title, version, **kwargs):
     """Utility to turn cornice web services into a Swagger-readable file.
     """
 
+    def get_tags_from_path(service):
+        return [service.path.split("/")[1]]
```

Comment:
Or I guess we should figure that a path should always start with a slash, like `/foo`?